helm/web/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
 appVersion: "1.0"
-description: A Helm chart for Kubernetes
+description: Simple Landing Page for ForgeOps
```

Comment:
Is it meant to verify completion of helm install or likely to have more purpose to showcase authn / authz ? [It may be good to indicate the purpose]

Reply:
Right now - just a landing page

Reply:
Thanks, Would like some visible cues such as landing page rather than kubectl get pods kind of things.